routing/message/handlers/passthrough/telemetry_handler.go

```diff
@@ -15,15 +15,18 @@ package passthrough
 import (
 	"github.com/ThreeDotsLabs/watermill"
 	"github.com/ThreeDotsLabs/watermill/message"
-	"github.com/eclipse-kanto/azure-connector/config"
-	"github.com/eclipse-kanto/azure-connector/routing"
+
 	"github.com/eclipse-kanto/suite-connector/connector"
 
+	"github.com/eclipse-kanto/azure-connector/config"
+	"github.com/eclipse-kanto/azure-connector/routing"
 	"github.com/eclipse-kanto/azure-connector/routing/message/handlers"
 )
 
 const (
 	telemetryHandlerName = "passthrough_telemetry_handler"
+
+	topicsEvent = "event/#,e/#,telemetry/#,t/#,command//+/res/#,c//+/s/#"
```

Comment:
`event/#` - by definition the topics should be in the format: event//${device-id} i.e. the filter should be: `event//+` Do we need to have such relaxed matching?

Reply:
The topics are taken from suite connector. They will be kept in this way.